.catalog-onboard-pipeline.yaml

```diff
@@ -0,0 +1,11 @@
+---
+apiVersion: v1
+offerings:
+  - name: watsonx-ai-SaaS
```

Comment:
You may need to change this name (in your ibm_catalog.json) as I don't think it will be approved. We have been using terminology like `deploy-arch-ibm-watsonx-ai-saas`

Reply:
done